apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/windows/GetStartedOverview.md

```diff
@@ -0,0 +1,13 @@
+Looking to build a Fluent experience on Windows? There are a few technologies you could use, depending on your needs.
+
+### WinUI 3
+
+WinUI is a user interface layer that contains native controls and styles for building Windows apps. It embodies Fluent Design, giving each Windows app the polished feel that customers expect.
+
+To build Fluent experiences on Windows using native frameworks, see [WinUI documentation](https://microsoft.github.io/microsoft-ui-xaml/).
+
+### Fluent UI React Native
+
+Fluent UI React Native is an [open-source](https://github.com/microsoft/fluentui-react-native) library built with JavaScript and [React Native components](https://microsoft.github.io/react-native-windows/). Use it to create Fluent experiences that work across Windows and across other platforms. It contains information about colors and typography, as well as custom controls and customizations.
+
+To build Fluent experiences on Windows using cross-platform frameworks, see [Cross-platform Get started page](#/get-started/crossplatform).
```

Comment:
@micahgodbolt - Jennifer has requested that this link should read "..., see [Get started with cross-platform](...)." See this bug if you have any questions. https://dev.azure.com/microsoftdesign/fluentui-native/_workitems/edit/7367

Reply:
```suggestion
To build Fluent experiences on Windows using cross-platform frameworks, see [Get started with cross-platform](#/get-started/crossplatform).
```